doc/notebooks/example_notebook.ipynb

```diff
@@ -17,12 +17,115 @@
  "nbformat": 4,
  "nbformat_minor": 2,
  "cells": [
+  {
+   "source": [
+    "# Try GRASS GIS in Jupyter Notebook with Python\n",
+    "\n",
+    "[<img src=\"grass_gis_logo.png\" alt=\"GRASS GIS\" style=\"width:200px;\"/>](https://grass.osgeo.org/)\n",
```

Comment:
Currently, there is no high-res logo in the source code, but you can try `../../man/grass_logo.png`.